Apply-For-Me-UI/src/pages/contact_us/components/ContactUsForm.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+const ContactUsForm = () => {
+
+    const formSubmit = (e) =>{
+        e.preventDefault();
+        alert("submitted");
+    }
+
+  return (
+    <div className="form">
+        <form onSubmit={formSubmit}>
+
+        
+
+
+                <div className="form-first-line">
+                    <div className="form-group first-line-1">
+                        <label>First Name</label>
+                        <input type="text" required></input>
+
+                    </div>
```

Comment:
- Work on making your design responsive, your design should stack when the contents are not looking okay as a row

###Your image

![image](https://user-images.githubusercontent.com/58771507/203979274-da9f74fa-ce9a-420d-9698-05f2ea8dd5c7.png)